### DISCUSSION TITLE
Mobile printing of ticket using label printer

### DISCUSSION THREAD

**Discussion by tengu1234** [alfio-event/alf.io#1340] [Q&A]
Hi there, 

I know you can set up an automatic check in system where a label printer will print name tags etc. I'm wondering if its possible to use the same label printer to print tickets?  We have a few events that will be catered toward seniors who don't necessairly use the internet or cell phones. It would be great if we can use that printer to print actual ticket they could take to present at the event.

**Comment by justusweigmann** [comment]
Hey @tengu1234

I support that idea. It would be great if that could be implemented somehow. Because we need printed out tickets for people who are not able to buy the tickets by themselves.
